### PR TITLE
Data Connector Description Update

### PR DESCRIPTION
**Pull request by v-prasadboke**
Required items, please complete
   
   Change(s):
   - Data Connector Description Update 

   Reason for Change(s):
   - Data Connector Description Update for the following Solutions
   - Oracle Database
   - Symantec Proxy SG 
   - Sophos XG Firewall

### CODE REVIEW

**Comment by v-rbajaj** [comment]
@v-prasadboke, please fix the validations

File: Solutions/OracleDatabaseAudit/Data Connectors/Connector_OracleDatabaseAudit.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
     "instructionSteps": [
         {
             "title": "",
-            "description": ">This data connector depends on a parser based on a Kusto Function to work as expected. [Follow these steps](https://aka.ms/sentinel-OracleDatabaseAudit-parser) to create the Kusto Functions alias, **OracleDatabaseAuditEvent**",
+            "description": "**NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected which is deployed as part of the solution. To view the function code in Log Analytics, open Log Analytics/Microsoft Sentinel Logs blade, click Functions and search for the alias Oracle Database Audit and load the function code or click [here](https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/OracleDatabaseAudit/Parsers/OracleDatabaseAuditEvent.txt), on the second line of the query, enter the hostname(s) of your Oracle Database Audit device(s) and any other unique identifiers for the logstream. The function usually takes 10-15 minutes to activate after solution installation/update.",
```
Comment (v-vdixit) [inline Solutions/OracleDatabaseAudit/Data Connectors/Connector_OracleDatabaseAudit.json:55]: Hi @v-prasadboke please update the note here and remove the part for hostname and other identifiers, keep the note inline with the parser instructions, thanks!

File: Solutions/Sophos XG Firewall/Package/createUiDefinition.json [hunk 2]
```diff
@@ -60,7 +60,7 @@
             "name": "dataconnectors1-text",
             "type": "Microsoft.Common.TextBlock",
             "options": {
-              "text": "This solution installs the data connector for ingesting logs from Sophos XG Firewall in syslog format. After installing the solution, configure and enable this data connector by following guidance in Manage solution view."
+              "text": "This Solution installs the data connector for Sophos XG Firewall. You can get Sophos XG Firewall Syslog data in your Microsoft Sentinel workspace. After installing the solution, configure and enable this data connector by following guidance in Manage solution view."
```
Comment (v-vdixit) [inline Solutions/Sophos XG Firewall/Package/createUiDefinition.json:63]: Hi @v-prasadboke, please discard these changes

File: Solutions/SymantecProxySG/Package/createUiDefinition.json [hunk 3]
```diff
@@ -60,14 +60,14 @@
...
+              "text": "This Solution installs the data connector for SymantecProxySG. You can get SymantecProxySG Syslog data in your Microsoft Sentinel workspace. After installing the solution, configure and enable this data connector by following guidance in Manage solution view."
             }
           },
           {
             "name": "dataconnectors-parser-text",
             "type": "Microsoft.Common.TextBlock",
             "options": {
-              "text": "The solution also installs a parser that transforms ingested data. The transformed logs can be accessed using the SymantecProxySG Kusto Function alias."
+              "text": "The Solution installs a parser that transforms the ingested data into Microsoft Sentinel normalized format. The normalized format enables better correlation of different types of data from different data sources to drive end-to-end outcomes seamlessly in security monitoring, hunting, incident investigation and response scenarios in Microsoft Sentinel."
```
Comment (v-vdixit) [inline Solutions/SymantecProxySG/Package/createUiDefinition.json:70]: Hi @v-prasadboke please remove the changes for data-connector and parser in this, thanks!

**Review by v-vdixit — APPROVED** [review]
Looks good!

**Pull request merged**